code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -43,6 +43,16 @@
 			ears.Insert(target_human, drop_if_replaced = FALSE)
 		else
 			mutantears = /obj/item/organ/internal/ears
+		var/obj/item/organ/internal/brain/current_brain = target_human.get_organ_by_type(/obj/item/organ/internal/brain)
+		if(current_brain)
+			current_brain.transform = current_brain.transform.Scale(0.8,0.8) //smaller brain
+	return ..()
+
+/datum/species/human/felinid/on_species_loss(mob/living/carbon/former_feline, datum/species/old_species, pref_load)
+	if(iscarbon(former_feline))
+		var/obj/item/organ/internal/brain/current_brain = former_feline.get_organ_by_type(/obj/item/organ/internal/brain)
+		if(current_brain)
+			current_brain.transform = current_brain.transform.Scale(1.25,1.25) //bigger brain
```

Comment:
Make the `0.8` a variable and make this `1 / that_variable` instead

Reply:
should be done